src/ui/components/VPNAnimatedRings.qml

```diff
@@ -64,6 +66,19 @@ Rectangle {
         property var ringXCenter: parent.width / 2
         property var ringYCenter: animatedRingsWrapper.yCenter
 
+        renderStrategy: Canvas.Threaded
+        renderTarget: Canvas.FramebufferObject
```

Comment:
Turns out: QT actually for every paint it put's this into a bitmap and copies it over to gpu manually. That's pretty slow. Since we have a gl context on every plattform, we can just enable the _FAST_ mode